docs/fr/features.md

```
@@ -327,7 +327,7 @@ Suivez ces étapes pour activer et configurer Antibot :
 
 1. Choisir un type de défi : décidez du mécanisme à utiliser (ex. [captcha](#__tabbed_3_3), [hcaptcha](#__tabbed_3_5), [javascript](#__tabbed_3_2)).
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)